src/Tools/ExternalAccess/OmniSharp/Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.csproj

```diff
@@ -0,0 +1,45 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.CodeAnalysis.ExternalAccess.OmniSharp</PackageId>
```

Comment:
Does this need to be marked as not going up to NuGet just like https://github.com/dotnet/roslyn/pull/53007?

Reply:
@jasonmalinowski still having a chat with those folks - I'm not 100% sure that is the solution as we've never had that marker yet we didn't use to upload those packages.

Reply:
Of note, all our current external access packages are on nuget: https://www.nuget.org/packages?q=Microsoft.CodeAnalysis.ExternalAccess

Reply:
Sigh, for 5.0.200 there was a goof up and all the packages were automatically published.

Reply:
@JoeRobich do we have a plan to unlist those? I know I saw some mail thread about unlisting due to a goof but not sure if it's more fallout of this or something else.

Reply:
In any case, so what's the recommendation for @333fred here, is there anything that we know that _is_ correct and can be put in this PR, or he should merge and we have a follow-up that cleans up his stuff along with everything else?

Reply:
Adding a property should not be necessary as we are responsible for publishing our own packages.

Reply:
Alright, marking as resolved then.